### PR TITLE
Fix some flaky unit tests

### PR DESCRIPTION
**Pull request by bgerstle**

### CODE REVIEW

File: WikipediaUnitTests/MWKImageFaceDetectionTests.m [hunk 1]
```diff
@@ -96,7 +96,7 @@ - (void)testDetectingFacesInFacelessImage {
         .then(^(NSArray* faces) {
             [self.image setNormalizedFaceBoundsFromFeatures:faces inImage:facelessImage];
         });
-    } timeout:WMFDefaultExpectationTimeout WMFExpectFromHere];
+    } timeout:WMFDefaultExpectationTimeout * 2 WMFExpectFromHere];
```
Comment (coreyfloyd) [inline WikipediaUnitTests/MWKImageFaceDetectionTests.m]: should this be:
`WMFDefaultExpectationTimeout * 2 * WMFExpectFromHere`

?

Comment (coreyfloyd) [inline WikipediaUnitTests/MWKImageFaceDetectionTests.m]: Also - should we just make a longer constant?

Comment (coreyfloyd) [inline WikipediaUnitTests/MWKImageFaceDetectionTests.m]: Or i guess I don't understand what `WMFExpectFromHere` is

Comment (bgerstle) [inline WikipediaUnitTests/MWKImageFaceDetectionTests.m]: > should this be:
WMFDefaultExpectationTimeout * 2 * WMFExpectFromHere

Nope, `WMFExpectFromHere` is a macro that passes the current method & line number to the method which is used to generate an expectation description.

> Also - should we just make a longer constant?

Eventually, yes. I'd rather move towards something like:

``` objc
typedef NS_ENUM(NSUInteger, WMFTestTimeout) {
  WMFTestTimeoutShort = 0,
  WMFTestTimeoutDefault,
  WMFTestTimeoutMedium,
  WMFTestTimeoutLong
};
WMFTimeout(NSUInteger timeoutLength) {
  float constant = 0.1;
  if (getenv("TRAVIS") != NULL) {
    constant = 1;
  }
  return powf(2, timeoutLength) * 0.1;
}
```

Comment (coreyfloyd) [inline WikipediaUnitTests/MWKImageFaceDetectionTests.m]: Ok - that Macro is… confusing… especially when it is sitting next to math… and being named like a constant.

We should look to do something to make it more idiomatic and clear in the future.

**Pull request merged**